### PR TITLE
Created listeners/mentionListener.js

### LINKED ISSUE #6: A mention embed
image
so a lot of bots have this mention -> info embed. Would be very useful for gathering all that information in one place (removes the need for a ping and uptime command)

### PR DESCRIPTION
**Pull request by sidtr0**
When a person types, `@Chronos`, it outputs this:

image

~~**Note:** `this.handler.prefix` does not work as well as it did in the previous `commands/about.js` file in my closed PR #11 so this has to be fixed.~~

### CODE REVIEW

File: src/listeners/mentionListener.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    if (message.mentions.users.first.id == this.client.id) {
+        let uptime = timeUtils.msToTime(this.client.uptime);
+
+        const embed = new MessageEmbed()
+        .setColor("#86F97C")
+        .setTitle("**Chronos**")
+        .setThumbnail(this.client.user.displayAvatarURL())
+        .setDescription("A bot that automatically manages slowmode in a channel, based on the activity in it.")
+        .addFields(
+            {name: "**Prefix**", value: `\`${this.handler.prefix}\``, inline: true},
```
Comment (sidtr0) [inline src/listeners/mentionListener.js]: This line needs to be fixed in a future PR

**Comment by sidtr0** [comment]
Haha so thanks to someone on the Discord server, the issue which I mentioned is fixed. I think it's all good and it can be merged now :+1:

File: src/listeners/mentionListener.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    });
+  }
+
+  exec(message) {
+
+    if (message.author.bot) {
+        return;
+    }
+
+    if (message.mentions.users.first.id == this.client.id) {
```
Comment (ItsAzaria) [inline src/listeners/mentionListener.js]: Almost perfect, it seems to react to any message not just mentions 
image

the issue is probably here. Try console logging those two values to see what they return

**Review by ItsAzaria — CHANGES_REQUESTED** [review]

File: src/listeners/mentionListener.js (hunk 2 above)
Comment (sidtr0) [reply src/listeners/mentionListener.js]: Huh, that's weird, I'll fix it.

**Review by ItsAzaria — APPROVED** [review]

**Pull request merged**